content/get-started.ts

```diff
@@ -110,17 +110,17 @@ const getStartedContent: GetStartedContent = {
     {
       id: 'step_1',
       details:
-        'Read "**Instructions for Vacating Misdemeanor and Gross Misdemeanor Convictions**": [CrRLJ 09.0300](https://www.courts.wa.gov/forms/?fa=forms.contribute&formID=38) from the Washington Courts website.',
+        'Read "**Instructions for Vacating Misdemeanors or Felony Convictions**": [CrRLJ 09.0300](https://www.courts.wa.gov/forms/?fa=forms.contribute&formID=38) / [CrRLJ 08.0930](https://www.courts.wa.gov/forms/?fa=forms.contribute&formID=38) from the Washington Courts website.',
```

Comment:
I think it should probably be "Misdemeanor or Felony convictions" not plural on the misdemeanor.

Reply:
Updated